ios/test-utils-core/Sources/utilities/HeadlessPlayerImpl.swift

```diff
@@ -1,6 +1,10 @@
 import JavaScriptCore
+
+#if SWIFT_PACKAGE
 import PlayerUI
 import PlayerUILogger
+#endif
```

Comment:
```suggestion
import PlayerUI
#if SWIFT_PACKAGE
import PlayerUILogger
#endif
```

We don't need to wrap `import PlayerUI` since that will be valid for cocoapods